service/bitbucket/openapi/README.md

```diff
@@ -0,0 +1,45 @@
+# Bitbucket OpenAPI
```

Comment:
Thank you for the investigation and create a valid Bitbucket Open API lib!

It's optional and I think it's ok to leave it in reviewdog repo for now, but it would be great to ...

1. Fix the issue in go-bitbucket lib.
or
2. Create a different repo. It's ok to create a repo under reviewdog org if you prefer.